Add audio feedback for correct and wrong answers in MultipleChoiceQuiz

Lys4nder opened the pull request:
Please help.

this path works: `await playAudio('/SpickApp/public/assets/audio/correct.mp3')`
this path doesn't `await playAudio('/assets/audio/correct.mp3')`

and I get a warning in code, with the working path:
`Files in the public directory are served at the root path.
Instead of /public/assets/audio/wrong.mp3, use /assets/audio/wrong.mp3.
Files in the public directory are served at the root path.
Instead of /public/assets/audio/correct.mp3, use /assets/audio/correct.mp3.`

But with that path, it doesn't work 🤔

Lys4nder committed:
Add audio feedback for correct and wrong answers in MultipleChoiceQuiz

Lys4nder committed:
code format

danielsofran commented on src/components/MultipleChoiceQuiz.tsx:
```diff
@@ -48,17 +48,30 @@ export const MultipleChoiceQuiz = (props: MultipleChoiceQuizProps) => {
       setSelected(prevSelected => [...prevSelected, index])
       if (index === correct) {
         setIsSuccess(true)
         await giveHapticFeedback(ImpactStyle.Heavy)
```
not on your commit but can we change this to ImpactStyle.Low or smth? it doesn't makes sense to have the same vibration on both correct or wrong

danielsofran commented on src/components/MultipleChoiceQuiz.tsx:
```diff
@@ -48,17 +48,30 @@ export const MultipleChoiceQuiz = (props: MultipleChoiceQuizProps) => {
     }
   }
 
+  const playAudio = async (audio: string) => {
```
please move this function and also the giveFeedback to a utils file.

danielsofran commented:
Ok so that's strange, because there is another path for assets in vite, they should be in src and not the public folder. that one is used only for assets for index.html, like the icon etc. I'll do a change

jibobek commented:
Try this. It should be same path, but it wont trigger that warning
```tsx
await playAudio(`${import.meta.env.BASE_URL}assets/audio/correct.mp3`)
```

danielsofran committed:
Fix audio source

- extract interactions in hooks/useInteractions
- fix publish job to deploy only from master (not possinble from other branches)
- improve format-check job to check the build as well
- add node modules caching to format check
- use --host by default for easier debuging
- add flags to bilingual title
- fix vibrations intensity (increased them)
- vibrate lower on correct answer
- ajust correct and wromng sounds not to be louder than the word recordings
- move download button in the menu
- remove test from menu (still available if url is manually changed)
